api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateNetworkOfferingCmd.java

```diff
@@ -297,7 +297,8 @@ public List<String> getSupportedServices() {
                         SourceNat.getName(),
                         PortForwarding.getName()));
             }
-            if (getNsxSupportsLbService()) {
+            if (getNsxSupportsLbService() || (provider != null && provider.equalsIgnoreCase("Netris") &&
```

Comment:
@Pearl1594 Please see if the methods below will help improve readability. Otherwise LGTM

private boolean isNsxWithLb() {
    return "Nsx".equalsIgnoreCase(provider) && getNsxSupportsLbService();
}

private boolean isNsxWithoutLb() {
    return "Nsx".equalsIgnoreCase(provider) && !getNsxSupportsLbService();
}

private boolean isNetrisNatted() {
    return "Netris".equalsIgnoreCase(provider) &&
           NetworkOffering.NetworkMode.NATTED.name().equalsIgnoreCase(getNetworkMode());
}

private boolean isNetrisRouted() {
    return "Netris".equalsIgnoreCase(provider) &&
           NetworkOffering.NetworkMode.ROUTED.name().equalsIgnoreCase(getNetworkMode());
}